assets/scss/components/_bounds.scss

```diff
@@ -13,12 +13,12 @@ padding, and sets a max-width.
 	box-sizing: border-box;
 	margin-left: auto;
 	margin-right: auto;
-	max-width: $bounds-wide;
+	max-width: $bounds;
```

Comment:
true, but if the goal is to use the bounds component feels like it should be an optional param there